src/theme.scss

```diff
@@ -0,0 +1,19 @@
+@import '~@playkit-js/playkit-js-ui';
+
+// border radius
+$roundness-1: 4px;
+$roundness-2: 8px;
+
+@mixin download-scrollbar {
```

Comment:
is this the stylized scrollbar component ? we already have it implemented in ui, in a class called Scrollable (which is wrapped around the contents that you want to scroll).
I think you can use it here, unless there's some difference in this scrollbar.